ehtim/features/rex.py

```diff
@@ -368,150 +408,196 @@ def plot_img(self, save_png=False):
 
         # position angle line
         pkloc = self.RingAngle1[0] * rad_to_pix
         lhloc = (self.RingAngle1[0] + self.RingAngle1[1]) * rad_to_pix
         rhloc = (self.RingAngle1[0] - self.RingAngle1[1]) * rad_to_pix
 
-        plt.axvline(x=pkloc,color=angcolor,linewidth=1)
-        plt.axvline(x=lhloc,color=angcolor,linewidth=1,linestyle=':')
-        plt.axvline(x=rhloc,color=angcolor,linewidth=1,linestyle=':')
+        plt.axvline(x=pkloc, color=angcolor, linewidth=1)
+        plt.axvline(x=lhloc, color=angcolor, linewidth=1, linestyle=':')
+        plt.axvline(x=rhloc, color=angcolor, linewidth=1, linestyle=':')
 
         # bright peak point
-        brightloc = (self.abspk_loc_rad, self.abspk_loc_ang) 
-        #plt.axvline(x=pkloc,color=pkcolor,linewidth=1)
-        plt.plot([self.abspk_loc_ang], [self.abspk_loc_rad], 'kx', mew=2, ms=6, color=pkcolor)
+        brightloc = (self.abspk_loc_rad, self.abspk_loc_ang)
+        # plt.axvline(x=pkloc,color=pkcolor,linewidth=1)
+        plt.plot([self.abspk_loc_ang], [self.abspk_loc_rad],
+                 'kx', mew=2, ms=6, color=pkcolor)
 
         # labels
         if xlabel:
-            plt.xlabel("$ \\theta $ ($^\circ$)", size=labelsize)
+            plt.xlabel(r"$ \\theta $ ($^\circ$)", size=labelsize)
         if ylabel:
-            plt.ylabel("$r$ ($\mu$as)", size=labelsize)
+            plt.ylabel(r"$r$ ($\mu$as)", size=labelsize)
 
         #xticklabels = np.arange(0,390,60)
-        xticklabels = np.arange(0,360,60)
-        xticks = (360/imarr.shape[1])*xticklabels
+        xticklabels = np.arange(0, 360, 60)
+        xticks = (360 / imarr.shape[1]) * xticklabels
 
-        yticks = np.floor(np.arange(0,imarr.shape[0],imarr.shape[0]/5)).astype(int)
-        yticklabels = ["%0.0f"%r for r in self.rs[yticks]]
+        yticks = np.floor(
+            np.arange(
+                0,
+                imarr.shape[0],
+                imarr.shape[0] /
+                5)).astype(int)
+        yticklabels = ["%0.0f" % r for r in self.rs[yticks]]
 
         if not xticklabel:
-            xticklabels=[]
+            xticklabels = []
         if not yticklabel:
-            yticklabels=[]
+            yticklabels = []
 
         plt.xticks(xticks, xticklabels)
         plt.yticks(yticks, yticklabels)
-        #plt.tick_params(axis='both',which='minor',length=6)
-        plt.tick_params(axis='both',which='major',length=6)
+        # plt.tick_params(axis='both',which='minor',length=6)
+        plt.tick_params(axis='both', which='major', length=6)
 
         if save_png:
             dirname = os.path.basename(os.path.dirname(self.imname))
             if not os.path.exists(POSTPROCDIR + '/' + dirname):
                 subprocess.call(['mkdir', POSTPROCDIR + '/' + dirname])
 
             basename = os.path.basename(self.imname)
-            fname = POSTPROCDIR + '/' +dirname+ '/' + basename[:-5] + '_unwrapped.png'
+            fname = POSTPROCDIR + '/' + dirname + \
```

Comment:
I suggested this as a format string some line above, but even if that is not the chosen way to go, I prefer the old version better in a single line, this is not very readable